plugin/plugins/mijia/__init__.py

```
@@ -559,6 +615,86 @@ async def get_cached_devices(self, refresh: bool = False, **_):
+        description="列出当前账号下所有米家智能场景，支持缓存",
+        input_schema={
+            "type": "object",
+            "properties": {
+                "home_id": {"type": "string", "description": "家庭ID，留空自动使用第一个"},
+                "refresh": {"type": "boolean", "description": "是否强制刷新缓存"}
+            },
+            "required": []
+        },
+        llm_result_fields=["message"]
+    )
+    async def list_scenes(self, home_id: str = None, refresh: bool = False, **_):
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)